assets/scss/local.scss

```diff
@@ -30,3 +30,17 @@
 .app-u-multiline-text {
   white-space: pre-line;
 }
+
+.app-u-button-as-link {
```

Comment:
New css class to style a button as a link (as used on the "Remove" button on the list of qualifications)
Discussed with @chrimesdev this morning 👍